CHANGELOG.md

```diff
@@ -88,6 +88,153 @@ If you are not using Nunjucks macros, update your HTML markup using the [back li
+      </ul>
+    </div>
+
+    <div class="nhsuk-do-dont-list">
+      <h3 class="nhsuk-do-dont-list__label">Don&#39;t</h3>
+      <ul class="nhsuk-list nhsuk-list--cross" role="list">
+        <li>
+  -       <svg class="nhsuk-icon nhsuk-icon__cross" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="34" height="34">
+  -         <path d="M17 18.5c-.4 0-.8-.1-1.1-.4l-10-10c-.6-.6-.6-1.6 0-2.1.6-.6 1.5-.6 2.1 0l10 10c.6.6.6 1.5 0 2.1-.3.3-.6.4-1 .4z" fill="#d5281b"></path>
+  -         <path d="M7 18.5c-.4 0-.8-.1-1.1-.4-.6-.6-.6-1.5 0-2.1l10-10c.6-.6 1.5-.6 2.1 0 .6.6.6 1.5 0 2.1l-10 10c-.3.3-.6.4-1 .4z" fill="#d5281b"></path>
+  -       </svg>
+  +       <svg class="nhsuk-icon nhsuk-icon__cross" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="34" height="34" focusable="false" aria-hidden="true">
+  +         <path d="M17 18.5c-.4 0-.8-.1-1.1-.4l-10-10c-.6-.6-.6-1.6 0-2.1.6-.6 1.5-.6 2.1 0l10 10c.6.6.6 1.5 0 2.1-.3.3-.6.4-1 .4z M7 18.5c-.4 0-.8-.1-1.1-.4-.6-.6-.6-1.5 0-2.1l10-10c.6-.6 1.5-.6 2.1 0 .6.6.6 1.5 0 2.1l-10 10c-.3.3-.6.4-1 .4z"/>
+  +       </svg>
+          do not burst a blister yourself
+        </li>
+      </ul>
+    </div>
+  ```
+
+- Header
+
+  If you are using search, update the search submit icon:
+
+  ```diff
+    <button class="nhsuk-header__search-submit" type="submit">
+  -   <svg class="nhsuk-icon nhsuk-icon__search" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+  -     <path d="M19.71 18.29l-4.11-4.1a7 7 0 1 0-1.41 1.41l4.1 4.11a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42zM5 10a5 5 0 1 1 5 5 5 5 0 0 1-5-5z"></path>
+  -   </svg>
+  -   <span class="nhsuk-u-visually-hidden">Search</span>
+  +   <svg class="nhsuk-icon nhsuk-icon__search" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="34" height="34" focusable="false" role="img" aria-label="Search">
+  +     <title>Search</title>
+  +     <path d="M19.71 18.29l-4.11-4.1a7 7 0 1 0-1.41 1.41l4.1 4.11a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42zM5 10a5 5 0 1 1 5 5 5 5 0 0 1-5-5z"></path>
+  +   </svg>
+    </button>
+  ```
+
+  If you are using an account link, update the user icon:
+
+  ```diff
+    <a class="nhsuk-header__account-link" href="#">
+  -   <svg class="nhsuk-icon nhsuk-icon__user" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+  -     <path d="M12 1a11 11 0 1 1 0 22 11 11 0 0 1 0-22Zm0 2a9 9 0 0 0-5 16.5V18a4 4 0 0 1 4-4h2a4 4 0 0 1 4 4v1.5A9 9 0 0 0 12 3Zm0 3a3.5 3.5 0 1 1-3.5 3.5A3.4 3.4 0 0 1 12 6Z"></path>
+  -   </svg>
+  +   <svg class="nhsuk-icon nhsuk-icon__user" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="34" height="34" focusable="false" aria-hidden="true">
+  +     <path d="M12 1a11 11 0 1 1 0 22 11 11 0 0 1 0-22Zm0 2a9 9 0 0 0-5 16.5V18a4 4 0 0 1 4-4h2a4 4 0 0 1 4 4v1.5A9 9 0 0 0 12 3Zm0 3a3.5 3.5 0 1 1-3.5 3.5A3.4 3.4 0 0 1 12 6Z"/>
+  +   </svg>
+      florence.nightingale@nhs.net
+    </a>
+  ```
```

Comment:
If we decide to include this change in v10, we won’t need to add this to the change log.

Reply:
Had a chat with @anandamaryon1 yesterday and he's going to review too

Mentioned in https://github.com/nhsuk/nhsuk-frontend/pull/1521#pullrequestreview-3115512731 but do we have context on where the icon changes have come from?

Got a bit more info in https://github.com/nhsuk/nhsuk-frontend/pull/1026 and https://github.com/nhsuk/nhsuk-frontend/pull/1027 but let's create or link to an issue